BackofficeBundle/Form/DataTransformer/ChoiceArrayToStringTransformer.php

```diff
@@ -0,0 +1,39 @@
+<?php
+
+namespace OpenOrchestra\BackofficeBundle\Form\DataTransformer;
+
+use Symfony\Component\Form\DataTransformerInterface;
+
+/**
+ * Class ChoiceArrayToStringTransformer
+ */
+class ChoiceArrayToStringTransformer implements DataTransformerInterface
+{
+    /**
+     * @param array|string $dataChoice
+     *
+     * @return string
+     */
+    public function transform($dataChoice)
+    {
+        if (null === $dataChoice || $dataChoice === array()) {
+            return "";
+        }
+
+        if (is_scalar($dataChoice)) {
```

Comment:
ChoiceArrayToStringTransformer is only used on ChoiceType and i don't think it can accept other thing than array or string.